h2o-py/h2o/explanation/_explain.py

```diff
@@ -1228,6 +1228,10 @@ def pd_multi_plot(
             plt.savefig(fname=save_plot_path)
         return decorate_plot_result(figure=fig)
 
+def _center(col):
+        norm_num = col[0]
+        for i in range(len(col)):
+            col[i] = col[i] - norm_num
```

Comment:
(I would consider moving this function inside the `ice_plot` function since it has a very general name and is used only there. )

If I understand that function correctly it could be vectorized as follows:
```suggestion
def _center(col):
    col[:] = col - col[0]
```